src/System.Net.Http/tests/StressTests/HttpStress/Program.cs

```diff
@@ -558,6 +604,19 @@ int Combine(int h1, int h2)
                 return ((int)rol5 + h1) ^ h2;
             }
         }
+        public int TaskNum { get; }
+
+        public HttpClient HttpClient { get; }
+
+        public string GetRandomSubstring(string input)
+        {
+            int offset = _random.Next(0, input.Length);
+            int length = _random.Next(0, input.Length - offset + 1);
+            return input.Substring(offset, length);
+        }
+
+        public Version GetRandomVersion(Version[] versions) =>
```

Comment:
This will distribute uniformly across supplied protocols. Unless we expect protocol versions other than v1.1 or v2, might make sense to just supply a percentage for v2 at the cli level?

Reply:
> might make sense to just supply a percentage for v2 at the cli level?

I'm fine with that, too, if you prefer.  Since this was merged, feel free to make that change subsequently.